plugin/buycourses/lang/spanish.php

```diff
@@ -23,6 +23,9 @@
 $strings['CancelPayout'] = "Cancelar Payout";
 $strings['ContinuePayout'] = "Continuar con el Pago";
 $strings['ProceedPayout'] = "Proceder con el Pago";
+$strings['TheActualPlatformCommissionIs'] = "La actual comisión de la plataforma es de %s, este porcentaje será descontado del total del precio del producto cuya diferencia será el monto base para aplicar estas comisiones.";

```

Comment:
Should be TheActualPlatformCommissionIsX
The "X" is very important. It is the way we remember (in code) that it goes with a variable